dace/runtime/include/dace/xilinx/stream.h

```diff
@@ -14,12 +14,12 @@ template <typename T, unsigned vector_length, unsigned capacity>
 class FIFO {
  public:
   FIFO() : stream_(capacity) {
-    #pragma HLS INLINE
-    #pragma HLS STREAM variable=stream_ depth=capacity
+#pragma HLS INLINE
+#pragma HLS STREAM variable = stream_ depth = capacity
```

Comment:
You broke the pragma formatting :-) clang-format [cannot handle this](https://stackoverflow.com/questions/48639518/tell-clang-format-to-ignore-pragmas).

Reply:
oh no! are there any other files clang-format killed?

Reply:
Only this one as far as I can see

Reply:
Fixed

Reply:
if you refresh and look at the diff, it should be gone (I double checked)